tests/tests.rs

```diff
@@ -168,12 +192,37 @@ where
 				.unwrap_or(false)
 		{
 			for test_type in test_types {
+				// load test config
+				let test_config_path = path.parent().unwrap().join("Config.toml");
+				let test_config = fs::read_to_string(&test_config_path);
+				let test_config = match test_config {
+					Err(err) => {
+						if err.kind() == io::ErrorKind::NotFound {
+							None
+						} else {
+							panic!("{}: {}", test_config_path.display(), err);
+						}
+					},
+					Ok(value) => Some(value)
+				};
+				let test_config = if let Some(test_config) = test_config {
+					toml::from_str(&test_config).unwrap()
+				} else {
+					TestConfig::default()
+				};
+
+				if test_config.nightly && !rustversion::cfg!(nightly) {
+					continue;
+				}
+
+				if !rustversion::cfg!(nightly) {}
```

Comment:
What does this `if` do?

Reply:
nothing. I wonder why clippy odes not warn about it.